### PR TITLE
Userstory11

### PR DESCRIPTION
**Pull request by rabe200**
[for US11](https://github.com/users/rabe200/projects/1?pane=issue&itemId=25297643)

### CODE REVIEW

File: pages/cards/dailycard/results/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default function Results() {
   const drawnCards = useStore((state) => state.drawnCards);
   const cardMoods = useStore((state) => state.cardMoods);
   const setLastCard = useStore((state) => state.setLastCard);
-  const stats = cardMoods.find((card) => card.name === currentCard.name);
+  // const stats = cardMoods.find((card) => card.name === currentCard.name);
```
Comment (ella-blaupause) [inline pages/cards/dailycard/results/index.js:10]: You can delete this

File: pages/cards/dailycard/results/index.js [hunk 2]
```diff
@@ -19,10 +19,17 @@ export default function Results() {
     return (
       <ul>
         <li>Results</li>
+        {/* <li>Date: {currentCard.date}</li> */}
```
Comment (ella-blaupause) [inline pages/cards/dailycard/results/index.js:22]: here too

File: pages/cards/history/index.js [hunk 3]
```diff
@@ -36,12 +70,57 @@ export default function History() {
...
+            </optgroup>
+          </select>
+          <button onClick={(event) => refreshList(event)}>refresh</button>
+        </form>
         <ul>
-          {drawnCards.map((card) => {
+          {displayedCards.map((card) => {
             return (
               <Fragment key={card.uuid}>
+                <hr></hr>
```
Comment (ella-blaupause) [inline pages/cards/history/index.js:107]: I think hr-tag self-closing

File: src/store/store.js [hunk 4]
```diff
@@ -67,18 +69,20 @@ export const useStore = createLocalStorageStore(
...
             mood: 0,
+            currentMood: 0,
             clicks: newEntry.clicks + 1,
-            avgMood: 0,
+            averageMood: 0,
           });
       set(() => ({ cardMoods: [newEntry].concat(filteredArray) }));
     },
-    setCardAvgMood: () => set((state) => ({})),
+    // setCardAvgMood: () => set((state) => ({})),
```
Comment (ella-blaupause) [inline src/store/store.js:85]: commented code!

**Review by ella-blaupause — APPROVED** [review]
Nice work💪💪💪

**Comment by rabe200** [comment]
thanks a lot !

**Review by ella-blaupause — APPROVED** [review]
LGTM👍

I'm looking forward to seeing your styling.🫣

**Pull request merged**